packages/core/src/shared/telemetry/util.ts

```diff
@@ -92,7 +92,7 @@ export async function getUserAgent(
     return pairs.join(' ')
 }
 
-type EnvType = 'cloud9' | 'cloud9-codecatalyst' | 'codecatalyst' | 'local' | 'ec2' | 'sagemaker' | 'test' | 'unknown'
+type EnvType = 'cloud9' | 'cloud9-codecatalyst' | 'codecatalyst' | 'local' | 'ec2' | 'sagemaker' | 'test' | 'wsl' |'unknown' 
```

Comment:
you'll want to remove the trailing space. I think this will cause linting to fail

Reply:
shouldnt formatting remove this automatically?